src/shared/server-config.toit

```diff
@@ -150,6 +152,9 @@ class ServerConfigSupabase extends ServerConfig implements supabase.ServerConfig
   to-service-json [--der-serializer] -> Map:
     return to-json --der-serializer=der-serializer
 
+  compute-cache-key_ -> string:
+    return base64.encode (sha1.sha1 host)
```

Comment:
Maybe just return the host here and do the base64 + sha1 in the shared `cache-key` method?

Reply:
done.